.travis.yml

```diff
@@ -79,3 +79,5 @@ env:
     - secure: "Le4CMhklfadi4aBQIEaEMbsFIB608GOvSHjVUxkDxkkUAVwl/Ov4Dni5d0Tn4e/xcxPkcm+pPg64dn0Jxzwx6XfWlxhWC10vYh+/GjpZW1znahtb/Gf9CNZOJJEy5LSeI7/uJ3LYcFd0FU0EJSerNeQJc5d8jmJH8UnuqObHOk29YD//XILiLRa1XALEimwXeQyGQePBmDTxPQQv1VLFjgfaJa5Xy55Us7AKTML2V7lhaeKCSEIp3x9liLAtnKlJhyXaXO/e4b3ZJTgXwYh+vENK1E2pxalpjBNPaJNkvtbsjFtYNXJoXca+hBVs5Sq1PCBhkEGxqFUsD8VLQd+MEXp4MYOF5fBhxIa3qOSjtuR+WmZ9G6fEysEBV6Y3F3D6HYWTpNkcHNXJCwdtOM+n92zNEBDIrufwzTPpyJXpoxZCCXrk3HHRdyDktvJYLrHdn1bM19mgYguesMZHTC5xMD6ifwdRoylmApjImXOvVxf2HdQiNvNLDqvaHgmYwNfl0+KbaVz+O2EDPCRnT5wOCpSeSUet47EPITdjr5OnTwLpOVaY+iSvn90EUB/8+ZU01TRYgc+6VNPHokLVjuiQJSrE4yTx/c2MnY9eRaOosVXngYfoS/L3XwDwZiQoeLZs04bScvxzGQIGCJ+CBzNPENtZ4AUh55Yl/vVNReZJeaY="
     # SPREAD_GOOGLE_KEY
     - secure: "dIA2HrartowFL2Gl5jXiVMd9hIJyIeummYwxeBL9MzO48E/BIJyIGHudEOo8oCnZ5a0yb8TqYgND2FCgJU1V5I2LyxH6T9kizHjtmIGgeM4qlEGKRlptb2v7DFkaHeW4Mpp4gLk8hYIeWyq9OR+SlK6f0Jj049LLKfQoX6GzTPug5+MMEQOJs55OJ6f6gvCv2o3oj6WFybaohMCO4GbNYQSPLwheyTSkT0efnW9QqTN0w62pDMqscVURO90/CUeZyCcXw2uOBegwPNTBoo/+4+nZsfSNeupV8wX4vVYL0ZFL6IO3mViDoZBD4SGTNF/9x8Lc1WeKm9HlELzy5krdLqsvdV/fQSWhBzwkdykKVA3Aae5dAMIGRt7e5bJaUg+/HdtOgA5jr+qey/c/BN11MyaSOMNPNGjRuv9NAcEjxoN2JkiDXfpA3lE9kjd7TBTexGe4RJGJLJjT9s8XxdKufBfruC/yhVGdVkRoc2tsAJPZ72Ds9qH0FH28zNFAgAitCLDfInjhPMPvZJhb3Bqx5P/0DE5zUbduE9kYK0iiZRJ4AaytQy+R4nJCXE42mWv5cxoE84opVqO9cBu1TPCC8gTRQFWpJt1rP+DvwjaFiswvptG8obxNpHmkhcItPGmRVN9P9Yjd9nHvegS83tsbrd2KOyMmCk3/1KWhLufisHE="
+    # GITHUB_API_KEY
+    - secure: "i6T6BYhtzsoBB41rRFW/vpFXMjOBQMb/N/c8H0IHtNTuam18WZiDFlydfZdzPrYxqlpk39yC2EZNqWB9+QOUCMMBWw/znXyXtQohXkEyKek7gv6Zsf1S+XZ6HeGetTtKYAc914S2/lbA9234rQd6RB7cuHO2cJEME01xucSPlB0zWZhFI5OoeLcwhA1zcGZIQDsfWYTVsAxHenKrDLbBnxgf4WCEJRyjcOQd3WCck+LW5e0yTTu9PbA81Ibr6b7tu1e/2mST1ZYrlUwN6IRsIjw7JfQCD4kfYXDSS/mbiIVEfN275pBgepAdj+bM/6fdZwaeFClzc+FUYWXFQ6FFyoXGvGuB1wOy8rXbnjc3euhiimBBfo8rJpnD63kzj4kaeo4Qar2s8bM4voxKdMmaXR3WKr0dFbMvdUBjS/HRGiwWngGmqhzKPLfzc3Qm4Ke1D8sA8pTzcu/kXOv1G3lH2lhUrgxgZ6DGceuoDM4ZwT4dhbUO+y4UhNLBtxKv5QT8tPbVxbOuWBZM87R1OHL5/O5pumTDH5M/UU0UW8g1UtBCLLUyiy1Re5UdNAkyG1di5NnIlDrdepYgpSXjW7tTcCt8eTmhU1cwzjoJ4YaoSGIkYZwD7ClirrbLJQ/+WezzpyfB5vSkGPOkkBPEHatlcrkc+LFY5cX1t+9fGgTCRAk="
```

Comment:
do we know how often these tokens expire?

Reply:
The test will stop working. It will never expire though, it may be deleted though.

Reply:
Fortunately the kind of permission this token needs are super minimal - just public RO repo access, essentially the same as anonymous access. The only reason why we need it at all is that travis IPs will hit the rate-limit from github because (apparently) a lot of people try to use anonymous API access to GH.